templates/email/bexley/waste/_council_reference.txt

```diff
@@ -0,0 +1 @@
+Your reference number is [% problem.id %].
```

Comment:
Presumably this needs a .html equivalent

Reply:
I think the HTML version was added in 14843278c9632b5725d5156840007fa663c53dd6, so was already in place before this pull request.